### ISSUE TITLE
Use already installed service

### ISSUE THREAD

**Issue by thiagogsr** [TablePlus/DBngin#3]
Hey guys, this tool looks awesome, is it work with the Postgres installed by homebrew?

**Comment by huyphams** [comment]
Yeah, we're working on that part. DBngin will be able to manage all brew services in the future release.

**Comment by hongshaowei** [comment]
Hi team, so for now already installed pgsql is still not able to work with?
Cause I saw the pic with homebrew below, so... not sure about it.

**Comment by huyphams** [comment]
It hasn't released yet @hongshaowei, the next update includes dark theme + homebrew. stay stunned.

**Comment by voltechs** [comment]
Has this been officially released? Asking for posterity (if folks come searching for this). I just updated to build 32. Looks like I can start and stop brew databases...? (❤️)
